controller/backup_controller.go

```diff
@@ -587,6 +591,61 @@ func (bc *BackupController) hasMonitor(backupName string) *engineapi.BackupMonit
 	return bc.monitors[backupName]
 }
 
+// getBackupVolumeRecurringJobs get recurring jobs in the volume labels and recurring jobs of groups in the volume labels
+func (bc *BackupController) getBackupVolumeRecurringJobs(volume *longhorn.Volume) string {
+	volumeRecurringJobSpecList := make(map[string]longhorn.RecordVolumeRecurringJob)
+
+	allRecurringJobs, err := bc.ds.ListRecurringJobsRO()
+	if err != nil {
+		bc.logger.WithError(err).Warn("Failed to list all recurring jobs")
+	}
```

Comment:
nit: Probably can improve the readability by
```suggestion
	allRecurringJobs, err := bc.ds.ListRecurringJobsRO()
	if err != nil {
		bc.logger.WithError(err).Warn("Failed to list all recurring jobs")
		return .......
	}
	
	volumeRecurringJobSpecList := make(map[string]longhorn.RecordVolumeRecurringJob)
	....
```

Reply:
Modified and returned error. Thanks